app/src/main/java/com/simplecity/amp_library/ui/presenters/PlayerPresenter.java

```diff
@@ -124,11 +129,15 @@ public void bindView(@NonNull PlayerView view) {
     }
 
 
-    private void refreshCurrentTimeText(long playbackTime) {
+    private void refreshTimeText(long playbackTime) {
         if (playbackTime != currentPlaybackTime) {
             PlayerView view = getView();
             if (view != null) {
                 view.currentTimeChanged(playbackTime);
+                if(SettingsManager.getInstance().preferRemainingTime())
```

Comment:
The formatting here is inconsistent with the rest of the app. Should have a space after `if`, and curly braces after the statement, i.e:
```
if (condition) {
    // Do something
```